packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -321,6 +321,26 @@ module.exports = {
         '/test_page/'
       ]
     },
+    'robots': {
+      host: 'https://developer.okta.com',
+      sitemap: "/docs-sitemap.xml",
+      policies: [
+        {
+          userAgent: '*',
```

Comment:
do we also need to set 'crawl-delay: 10'? the existing `robots.txt` file has it

Reply:
fixed: https://60084b4343761827e3c7e7ad--reverent-murdock-829d24.netlify.app/robots.txt